Contribute/index.html

```diff
@@ -22,11 +22,13 @@ <h3>We welcome contributions to all aspects of the seL4 ecosystem:</h3>
   <li class="list-tick">Code and documentation contributions via pull requests on
     <a href="https://github.com/seL4">GitHub</a>. You should be aware of the seL4
     <a href="{% link Contribute/conventions.html %}">coding conventions and
-    requirements</a> and the <a href="dco.html">developer certificate of origin</a>.
-    See <a href="#code">more details below.</a>
+    requirements</a> and the <a href="dco.html">developer certificate of
+    origin</a>.  See <a href="#code">more details below.</a> For the specific
+    case of contributing a platform port, also read
+    the <a href="platforms.html">platform contributions requirements</a>.
```

Comment:
Good idea to mention it, but I'll move the platform ports further below to the details section where we talk about other specific contributions (kernel, docs, etc)